### PR TITLE
Make goal and fk variable names depend on goal indices

### PR DESCRIPTION
**Pull request by casparvv**
This seems to fix issue #38.

### CODE REVIEW

File: plannerbenchmark/postProcessing/helpers.py [hunk 1]
```diff
@@ -40,6 +40,7 @@ def createMetricsFromNames(
         Returns a list of all metrics for which the name was specified.
     """
     metrics = []
+    goal_indices = experiment.primeGoal().indices()
     m = experiment.obstacles()[0].dim()
```
Comment (maxspahn) [inline plannerbenchmark/postProcessing/helpers.py:44]: So we still need 'm' then? If so, could we give it a more meaningful name, such as dimension_obstacle?

**Review by maxspahn — COMMENTED** [review]
Great! I'll check it on Monday

File: plannerbenchmark/postProcessing/helpers.py (hunk 1 above)
Comment (casparvv) [reply plannerbenchmark/postProcessing/helpers.py]: Indeed, in ``helpers.py`` itself, we don't really need `m`. However, it is passed along to the parameters of the metric classes, where it is still used.
Changed the names in another commit.
Should we do the same for the `n` variable? Could be dimension_configuration.

https://github.com/maxspahn/localPlannerBench/blob/77dfe2698e27e63690b7d709fad62932eaac7eaf/plannerbenchmark/postProcessing/helpers.py#L44

**Comment by casparvv** [comment]
I propose to make the time2Goal metric depend on the goal dimension instead of the obstacle dimension, will add this to the PR.
Otherwise, shape mismatch can occur in calculating the distance between the fks and the goal if the dimension of the goal is 2, but an obstacle dimension of three is used.
Other metrics might also need to be made goal dependent.

File: plannerbenchmark/postProcessing/helpers.py (hunk 1 above)
Comment (maxspahn) [reply plannerbenchmark/postProcessing/helpers.py]: Let's keep this `n` for the moment as it is not crucial for the time being. 
As there will be an update to motionplanningscenes, we should update all names.

**Pull request merged**